R/colour-text.r

```diff
@@ -5,9 +8,19 @@ testthat_colours <- list(
   error = crayon::red
 )
 
-colourise <- function(text, as = c("passed", "skipped", "error")) {
+#' @export
+colourise <- function(text, as = c("passed", "skipped", "error"), fg, bg) {
+  fg_warn <- function() warning("The fg parameter to testthat::colourise is deprecated.", call. = FALSE)
+  if (!missing(fg)) fg_warn()
+  if (any(is.element(as, allowed_colors))) {
+    fg_warn()
+    return(eval.parent(sys.call()[1:2]))
+  }
+  if (!missing(bg)) warning("The bg parameter to testthat::colourise is deprecated.", call. = FALSE)
```

Comment:
I think we should add a test for the compatibility behavior. We could mock `getOption` or just set the corresponding option to `FALSE` for the test.